ambry-store/src/test/java/com.github.ambry.store/CuratedLogIndexState.java

```diff
@@ -57,12 +57,13 @@
 class CuratedLogIndexState {
   private static final byte[] RECOVERY_INFO = new byte[100];
   // setupTestState() is coupled to these numbers. Changing them *will* cause setting test state or tests to fail.
-  private static final long LOG_CAPACITY = 10000;
+  private static final long LOG_CAPACITY = 20000;
   private static final long SEGMENT_CAPACITY = 2000;
   private static final int MAX_IN_MEM_ELEMENTS = 5;
   private static final long HARD_DELETE_START_OFFSET = 11;
   private static final long HARD_DELETE_LAST_PART_SIZE = 13;
 
+  static final long DELAY_BETWEEN_LAST_MODIFIED_TIMES_MS = 10 * Time.MsPerSec;
```

Comment:
This is exposed to `BlobStoreStatsTest` for the purpose of testing at meaningful time points.